odl/datasets/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Access to real or simulated datasets for use with ODL."""
+
+from __future__ import absolute_import
+
+
+__all__ = ('ray_transform',)
+
+from . import ray_transform
```

Comment:
But it's still confined to the `datasets` package, so it won't pollute the top-level namespace. Anyway, I don't have a strong opinion here. Given that it's easier to later add it to `__all__` than to remove it again, we can keep it the way it is here.